src/crab/array_domain.cpp

```
@@ -275,17 +152,17 @@ std::vector<Cell> offset_map_t::get_overlap_cells_symbolic_offset(const Interval
         // For efficiency, we assume it overlaps with all.
         Cell largest_cell;
         for (const Cell& c : o_cells) {
```

Cppcheck (2.19.0)

[style] The function 'udiv' is never used.

(unusedFunction)